components/RadioGroupDefault/RadioGroupDefault.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { RadioGroupDefault, RadioGroupDefaultItem } from './RadioGroupDefault';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { LabelText } from '../LabelText/LabelText';
+
+describe('RadioGroupDefault', () => {
+  beforeEach(() => {
+    render(
+      <RadioGroupDefault defaultValue="all" required>
+        <div className="flex items-center space-x-2">
+          <RadioGroupDefaultItem value="all" id="all" />
+          <LabelText htmlFor="all">All users</LabelText>
+        </div>
+      </RadioGroupDefault>,
+    );
+  });
```

Comment:
Can you add a new line here please? It feels a bit cluttered here.

Reply:
Yup, good call.

Reply:
Will be reflected in the next commit.